app/src/main/java/com/jerboa/ui/components/post/composables/PostComposables.kt

```diff
@@ -141,9 +146,36 @@ fun CreateEditPostBody(
             PictrsUrlImage(
                 url = url,
                 blur = false,
+                contentDescription = altText,
+            )
+
+            /**
+             * Alt text field. Only show if its a url image.
+             */
+            OutlinedTextField(
```

Comment:
Alt text is probably a recent feature? You might wanna add a feature flag check to hide these for older Lemmy Versions

Reply:
Ah forgot about that, good call.

Reply:
K I added them for hiding posts, alt_text, and custom_thumbnails, but all are using the `FF.hidePost` since `FF.altText` doesn't exist (Its nbd to me).

Reply:
Yeah I kinda forgot about altText when I generated the Types. I'll add one but yeah it doesn't really matter

Reply:
Cool, ya its not vital for this anyway.